nixos/tests/containers-imperative.nix

```diff
@@ -11,40 +11,11 @@ import ./make-test.nix ({ pkgs, ...} : {
     { imports = [ ../modules/installer/cd-dvd/channel.nix ];
       virtualisation.writableStore = true;
       virtualisation.memorySize = 768;
-
-      containers.webserver =
-        { privateNetwork = true;
-          hostAddress = "10.231.136.1";
-          localAddress = "10.231.136.2";
-          config =
-            { services.httpd.enable = true;
-              services.httpd.adminAddr = "foo@example.org";
-              networking.firewall.allowedTCPPorts = [ 80 ];
-              networking.firewall.allowPing = true;
-            };
-        };
-
       virtualisation.pathsInNixDB = [ pkgs.stdenv ];
     };
 
   testScript =
```

Comment:
This test fails for me because there is no no nameserver available:

```
machine# error checking the existence of http://tarballs.nixos.org/sha256/13i04dkd709f0p5f2413sf2y9321pfi4y85ynf8wih6ryphnbk9x:
machine# curl: (6) name lookup timed out
machine# 
machine# trying http://www.docbook.org/xml/5.0/docbook-5.0.zip
machine# Warning: Transient problem: timeout Will retry in 1 seconds. 3 retries left.
machine# Warning: Transient problem: timeout Will retry in 2 seconds. 2 retries left.
machine# Warning: Transient problem: timeout Will retry in 4 seconds. 1 retries left.
machine# curl: (6) Couldn't resolve host 'www.docbook.org'
machine# error: cannot download docbook-5.0.zip from any mirror
machine# builder for ‘/nix/store/mh8hylyfkrmbhjbpz4xkz8f8lx3hcbvc-docbook-5.0.zip.drv’ failed with exit code 1
```